src/api/app/controllers/concerns/webui/requests_filter.rb

```diff
@@ -7,7 +7,7 @@ def filter_requests
     @selected_filter = { states: %w[new review], action_types: [], creators: [],
                          priorities: [], staging_projects: [], reviewers: [],
                          project_names: [], created_at_from: nil, created_at_to: nil,
-                         involvement: [], search: nil, package_names: [] }.with_indifferent_access
+                         involvement: %w[incoming outgoing review], search: nil, package_names: [] }.with_indifferent_access
```

Comment:
This change in the defaults makes the feature tests fail for:
- `spec/features/beta/webui/my_requests_spec.rb`
- `spec/features/beta/webui/project_requests_spec.rb`
- `spec/features/beta/webui/package_requests_spec.rb`